Utilities/IntervalFiles/ComputeIntervalBamStats.wdl

```diff
@@ -0,0 +1,306 @@
+version 1.0
+
+# For computing stats/distributions over various interval files for BAM files
+workflow ComputeIntervalBamStats {
+    input {
+        String input_name
+        File input_bam
+        File input_bam_index
+        String experiment = ""
+
+        File ref_fasta
+        File ref_index
+        File ref_dict
+
+        Array[File] interval_files
```

Comment:
Is there a guarantee taht interval files and interval names will be in the correct order? Or is this a manual check? Or do the names not matter so much?

Reply:
The zip in the WDL should guarantee the arrays get paired up exactly in the same order as input. Of course there's no way to prevent users from mislabeling their interval files in an order different than the one they gave (outside of using some custom structs for input, but those get a little messy to format for the input IMO).